src/v2/view-builder/ViewFactory.js

```diff
@@ -91,8 +93,11 @@ const VIEWS_MAPPING = {
     password: ChallengeAuthenticatorPasswordView,
     webauthn: RequiredFactorWebauthnView,
     'security_key': ChallengeWebauthnView,
-    phone: null,
     'security_question': ChallengeAuthenticatorSecurityQuestion,
+    phone: ChallengeAuthenticatorPhoneView,
+  },
+  'authenticator-verification-data': {
+    phone: ChallengeAuthenticatorPhoneOptionsView,
```

Comment:
Given `options` has been used in other place, would suggest to name it `ChallengeAuthenticatorDataPhoneView`